cluster/libexec/wwinit/70-iptables.init

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```

Comment:
We should just warn the user that firewalld, nft, or iptables is running and that it might needs to be modified, or actually add the port 80 file for them. We shouldn't disable a firewall on a user especially on ALL and MASTER.

Reply:
Opened #264 to track this.